src/Swarm/TUI/View.hs

```diff
@@ -132,17 +136,43 @@ drawNewGameMenuUI (l :| ls) =
     . centerLayer
     $ hBox
       [ vBox
-          [ withAttr robotAttr . txt $ breadcrumbs ls
+          [ withAttr boldAttr . txt $ breadcrumbs ls
           , txt " "
           , vLimit 20 . hLimit 35
-              . BL.renderList (const drawScenarioItem) True
+              . BL.renderList (const $ padRight Max . drawScenarioItem) True
               $ l
           ]
       , padLeft (Pad 5) (maybe (txt "") (drawDescription . snd) (BL.listSelectedElement l))
       ]
  where
-  drawScenarioItem (SISingle s) = padRight Max . txt $ s ^. scenarioName
-  drawScenarioItem (SICollection nm _) = padRight Max (txt nm) <+> withAttr robotAttr (txt ">")
+  drawScenarioItem (SISingle s si) = padRight (Pad 1) (drawStatusInfo s si) <+> txt (s ^. scenarioName)
+  drawScenarioItem (SICollection nm _) = padRight (Pad 1) (withAttr boldAttr $ txt " > ") <+> txt nm
+  drawStatusInfo s si = case s ^. scenarioObjectives of
+    [] -> txt "   "
+    _ -> case si ^. scenarioBest of
+      NotStarted -> txt " o "
+      InProgress {} -> withAttr yellowAttr $ txt "..."
+      Complete {} -> withAttr greenAttr $ txt " ✓ "
+
+  describeStatus = \case
+    NotStarted -> txt "none"
+    InProgress _s e _t ->
+      withAttr yellowAttr . vBox $
+        [ txt "in progress"
+        , txt $ "(played for " <> formatTimeDiff e <> ")"
+        ]
+    Complete _s e t ->
+      withAttr greenAttr . vBox $
+        [ txt $ "completed in " <> formatTimeDiff e
+        , hBox
+            [ txt "("
+            , drawTime t True
+            , txt " ticks)"
+            ]
+        ]
```

Comment:
I just played with it and realized that the best time in ticks and real-time can be different. :confused: 

Maybe the best thing is to simply have a _best time_ and _best ticks_?
The UI could collapse that to one _best_ if they are the same and otherwise split it into two.

The `updateScenarioInfoOnQuit` would also need to be changed.

Reply:
OK, this will probably be rare, but I think it looks good this way:

|Same|Different|
|-|-|
|![image](https://user-images.githubusercontent.com/44544735/187264180-5b6b9570-eab7-451c-bef4-75b3d504e86e.png)|![Screenshot from 2022-08-29 19-39-08](https://user-images.githubusercontent.com/44544735/187264259-52c50227-0cf0-4a3b-b5d5-1b4ee88a691e.png)|


What do you think, @byorgey? :slightly_smiling_face:

Reply:
Well, ultimately I don't know if I want to give people an incentive to complete scenarios in the fewest ticks possible, since that just gives you an incentive to slow the game way down so you can type a lot on each tick, and that seems boring. :yawning_face:   But I think it is fine for now and we can maybe refine it later.

Reply:
To expand on the above comment a bit:
- Measuring *fewest ticks* incentivizes players to slow the game down a lot.  This is boring.
- Measuring *fastest time* incentivizes players to speed the game up a lot.  This favors players with beefy computers.
- So I think a good compromise for speedruns is to measure fastest time, but forbid adjusting the tick speed.
    - Eventually (but not in this PR) I'd like to have it track whether you adjusted the tick speed at all, and have a best time where you didn't adjust the tick speed, and a separate best time where you did adjust the tick speed.  Maybe also a flag `--speedrun` that disables adjusting the tick speed.

Reply:
Well for me it was just about getting the best possible tick count for challenges which I can solve with one command/algorithm.

For move its just 3, but in later challenges I will probably care about the number of digits. 😄

Given that tick clock almost matches seconds, it’s just a way to not count pausing e.g. when reading the goal. 🤷

Reply:
Oh, I see, that's a good point.  So you could write some code in a file and then start it with `--run` to get the fewest ticks.  I guess that's not actually so boring after all. :smile: